net/autossh/src/etc/rc.d/opnsense-autossh

```diff
@@ -0,0 +1,182 @@
+#!/bin/sh
+#
+# $FreeBSD$
+#
+
+# PROVIDE: autossh
+
+# Add the following /etc/rc.conf.d/autossh entry to enable autossh:
+# autossh_enable="YES"
+
+
+. /etc/rc.subr
+
+name=autossh
+rcvar=${name}_enable
```

Comment:
this will eventually clash with freebsd when an rc.d script is added in the package: rename var or maybe add a FreeBSD rc.d script?